modules/internal/DefaultRectangular.chpl

```diff
@@ -303,8 +303,10 @@ module DefaultRectangular {
       proc anyStridable(rangeTuple, param i: int = 1) param
         return if i == rangeTuple.size then rangeTuple(i).stridable
                else rangeTuple(i).stridable || anyStridable(rangeTuple, i+1);
-  
-      chpl__testPar("default rectangular domain follower invoked on ":string_rec, followThis);
+
+      if chpl__testParFlag then
+        chpl__testPar("default rectangular domain follower invoked on ":string_rec, followThis);
```

Comment:
Since this call won't occur by default, is it safe to remove the cast? If not if seems like the cast should also be added to the calls in BlockDist.chpl